scripts/format_arpa_lm.py

```diff
@@ -44,6 +46,9 @@
 ngram_order = int(f.readline())
 f.close()
 
+# set the memory restriction for "sort"
+sort_mem_opt = ("--buffer-size={0} ".format(args.max_memory))
```

Comment:
Sorry I made a mistake here. Will modify it soon.
Ke

On Fri, Aug 12, 2016 at 12:28 AM, Wang Jian <notifications@github.com>
wrote:

> In scripts/format_arpa_lm.py
> <https://github.com/danpovey/pocolm/pull/46#discussion_r74540270>:
>
> > @@ -44,6 +46,9 @@
> >  ngram_order = int(f.readline())
> >  f.close()
> >
> > +# set the memory restriction for "sort"
> > +sort_mem_opt = ("--buffer-size={0} ".format(args.max_memory))
>
> when args.max_memory == '', the sort_mem_opt should be '--buffer-size= '.
> I'm not sure whether sort would complain If we pass this sort_mem_opt to it.
> Is the magic of outter parentheses on the right side makes the
> sort_mem_opt to empty?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/danpovey/pocolm/pull/46/files/c42a9232ccf73be3005330a3fb811af9c2f523e9#r74540270>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ANVxSpniPy_TNegw7ep6OTf7pSA1CVC1ks5qe_ZTgaJpZM4JibxT>
> .
>



-- 
Ke Li
Dept. of Electrical and Computer Engineering
Johns Hopkins University
Email: kli26@jhu.edu